packages/mobile/src/home/saga.ts

```diff
@@ -37,7 +48,25 @@ export function* refreshBalancesWithLoadingSaga() {
   )
 }
 
-export function* watchRefreshBalances() {
+function* autoRefreshSaga() {
+  while (true) {
+    yield delay(10 * 1000) // sleep 10 seconds
+    if (yield select(shouldUpdateBalance)) {
+      refreshBalances()
+    }
+  }
+}
+
+function* autoRefreshWatcher() {
+  while (yield take(Actions.START_BALANCE_AUTOREFRESH)) {
```

Comment:
This is cool but what's the scenario when we enable / disable the autoRefresh feature?

Reply:
The autorefresh is disabled in line 64 https://github.com/celo-org/celo-monorepo/pull/109/files/bb240961e1cb2323e3c1cae4e2406f63606942dd#diff-254ba29c34bf91f00db0a1111e3b61c8R64